xsd_validate/xsd_validate_activity.go

```diff
@@ -0,0 +1,75 @@
+package xsdvalidate
```

Comment:
Here's one thought about package naming. I'm using `go doc` to inspect the public API of `xsdvalidate`.

```
$ go doc github.com/artefactual-sdps/temporal-activities/xsd_validate
package xsdvalidate // import "github.com/artefactual-sdps/temporal-activities/xsd_validate"

const XSDValidateActivityName = "xsd-validate-activity"
type XSDValidateActivity struct{}
    func NewXSDValidateActivity() *XSDValidateActivity
type XSDValidateActivityParams struct{ ... }
type XSDValidateActivityResult struct{ ... }
```

*(you can use `go doc -all` to include more details)*

Using `bagit` and `archive` as examples of packages that contain activities for their respective domains, renaming this package to `xml` could create an useful container for all XML-related activities, including validation and future XML manipulations, e.g.:

```
package xml // import "github.com/artefactual-sdps/temporal-activities/xml"

const ValidateActivityName = "validate-activity"
type ValidateActivity struct{}
    func NewValidateActivity() *ValidateActivity
type ValidateActivityParams struct{ ... }
type ValidateActivityResult struct{ ... }
```

Reply:
Sounds good!

Reply:
(Done!)

Reply:
Oops... fixed!